src/imp/noop_imp.js

```diff
@@ -1,18 +1,25 @@
+function NoopSpanContextImp() {
+}
+NoopSpanContextImp.prototype.setBaggageItem = function (key, value) {
+}
+NoopSpanContextImp.prototype.getBaggageItem = function (key) {
+}
+
 function NoopSpanImp() {
 }
 NoopSpanImp.prototype.tracer = function () {
     return null;
 }
+NoopSpanImp.prototype.context = function () {
+    // XXX what's the deal here? Why can tracer() return null?
```

Comment:
note: I left this file in an unfinished state as I wasn't sure I understood how it's meant to fit in.